software/contrib/quantizer.py

```diff
@@ -0,0 +1,496 @@
+## Equal-temperment quantizer for the EuroPi
+#
+#  Features configurable intervals for multiple outputs and customizable scale
+#
+#  \author Chris Iverach-Brereton <ve4cib@gmail.com>
+#  \date   2023-02-12
+
+from europi import *
+from europi_script import EuroPiScript
+import time
+
+## 1.0V/O is the Eurorack/Moog standard, but Buchla uses 1.2V/O
+#
+#  Just in case someone needs Buchla compatibility, this is defined
+#  but nobody is likely to need this
+VOLTS_PER_OCTAVE = 1.0
```

Comment:
The three options I see are:

- we merge the Quantizer as-is, and it can continue using the existing code, oblivious to the changes in #220
- we merge the Quantizer now, and once #220 is merged I/someone can update it to use the new config system
- we hold off merging in the Quantizer until after #220 is merged, I rebase, and update this merge request

Reply:
Definitely option 1 or 2. I just wanted to make you aware, in case it was something you were interested in.

Reply:
I'll keep an eye on what changes and see about submitting some updates.  The 3 new scripts I've submitted have some code copied between them I'd like to optimize out into some sort of shared library eventually, too.

Reply:
I'll keep an eye on what changes and see about submitting some updates. The 3 new scripts I've submitted have some code copied between them I'd like to optimize out into some sort of shared library eventually, too.